cassandra/__init__.py

```diff
@@ -744,9 +744,3 @@ def __init__(self, msg, excs=[]):
         if excs:
             complete_msg += ("\nThe following exceptions were observed: \n - " + '\n - '.join(str(e) for e in excs))
         Exception.__init__(self, complete_msg)
-
-class VectorDeserializationFailure(DriverException):
-    """
-    The driver was unable to deserialize a given vector
-    """
-    pass
```

Comment:
Added in PYTHON-1371 to indicate an attempt to decode a vector of an unsupported subtype.  With the other changes in this PR this exception is now completely unnecessary.